test/boost/sstable_datafile_test.cc

```diff
@@ -2968,7 +2968,7 @@ SEASTAR_TEST_CASE(compaction_with_fully_expired_table) {
         expired = get_fully_expired_sstables(*cf, ssts, gc_clock::now());
         BOOST_REQUIRE(expired.size() == 0);
 
-        auto ret = compact_sstables(sstables::compaction_descriptor(ssts, cf->get_sstable_set(), default_priority_class()), *cf, sst_gen).get0();
+        auto ret = compact_sstables(sstables::compaction_descriptor(ssts, cf->get_sstable_set_copy(), default_priority_class()), *cf, sst_gen).get0();
```

Comment:
Really, the code churn is unwarranted. get_sstable_set() should maintain its name and semantics. Only the implementation should be made more efficient.

Reply:
There shouldn't be problems with code churn now, almost all names and semantics changes have been removed in later versions